index.js

```diff
@@ -0,0 +1,292 @@
+const NOTES_DOM = {
+    notes_board:document.querySelector("#tasksBoard"),
```

Comment:
use camelCase for variables/fields so it will be NOTES_DOM.notesBoard

UPPER_CASE_WITH_UNDERSCORE is ok for consts